### PR TITLE
assessment response

### PR DESCRIPTION
**Pull request by faetea**
referred to my class notes
ran out of time

### CODE REVIEW

File: lib/assessment.js [hunk 1]
```diff
@@ -4,18 +4,35 @@
...
-var min = function min() {
-
+var min = function min(nums) {
+  var minValue = nums[0];
+  for (var i = 1; i < nums.length; i++) {
+    if (nums[i] < minValue) {
+      minValue = nums[i];
+    }
+  }
+  return minValue;
```
Comment (GA-MEB) [inline lib/assessment.js:14]: Yep, bingo.

File: lib/assessment.js [hunk 2]
```diff
@@ -4,18 +4,35 @@
...
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-var sum = function sum() {
-
+var sum = function sum(aryAdd) {
+  for (var i = 0; i < aryAdd.length; i++) {
+    aryAdd[i] += 1;
+  }
```
Comment (GA-MEB) [inline lib/assessment.js:32]: On the right track but not quite there. Consider the solution below.

``` javascript
var sum = function(){
  var total = 0;
  for (var i = 0; i < arguments.length; i++) {
    total += arguments[i];
  }
  return total;
};
```

**Comment by GA-MEB** [comment]
If you have a couple minutes this coming weekend, post project, it might be worth giving this another shot. Let me know if you'd like my feedback on your work!

**Comment by faetea** [comment]
Matt,
I would like your feedback, yes.  I just don't know what questions to ask
most of the time.  I am planning on looking back over all of the
assessments, I just haven't done so yet.

Allie

**Pull request closed without merging**